web/templates/settings_overlay.hbs

```diff
@@ -35,7 +35,7 @@
                     {{/unless}}
                     <li class="sidebar-item" tabindex="0" data-section="alert-words">
                         <i class="sidebar-item-icon fa fa-book" aria-hidden="true"></i>
-                        <div class="text">{{t "Alert words" }}</div>
```

Comment:
I think we'll want to change the URL hash for this tab of the personal settings overlay with these updates as well. See #26110 for when the "Display settings" tab was changed to "Preferences".